torchrl/objectives/redq.py

```diff
@@ -181,16 +180,9 @@ def forward(self, tensordict: TensorDictBase) -> TensorDictBase:
                 tensordict_actor,
                 actor_params,
             )
-            if isinstance(self.actor_network, TensorDictSequential):
-                sample_key = self.actor_network[-1].sample_out_key[0]
-                tensordict_actor_dist = self.actor_network[-1].build_dist_from_params(
-                    td_params
-                )
-            else:
-                sample_key = self.actor_network.sample_out_key[0]
-                tensordict_actor_dist = self.actor_network.build_dist_from_params(
-                    td_params
-                )
```

Comment:
I think it's safe to get rid of that, but can I double check that this assumption is correct?